index.js

```
@@ -193,6 +193,39 @@ module.exports = function drivex(driver, wd) {
       }, timeout || 5000).then(function() {
         return keyFound;
       });
+    },
+    /**
+     *validateText validates the text for a WebElement
+     * @param locator
+     * @param parentWebElement (optional)
+     * @param expected text
+     * @returns {*}
+     */
+    validateText: function (locator, parentWebElement, expectedText) {
+      return methods.find(locator, parentWebElement,expectedText).getText().then(function (actual) {
+        if(actual === expectedText){
+          return true;
+          } else {
+          return false;
+        }
+       });
+      },
+    /**
+     *validateAttributeValue validates the attribute for a WebElement
+     * @param locator
+     * @param parentWebElement (optional)
+     * @param attribute value
+     * @param expected text
+     * @returns {*}
+     */
+    validateAttributeValue: function (locator, parentWebElement,attribute, expectedText) {
+      return methods.find(locator, parentWebElement,expectedText).getAttribute(attribute).then(function (actual) {
+        if(actual === expectedText){
```

Instead of resolving as "false" we should throw an error in order to be consistent with other similar functions.